### ISSUE TITLE
[Bug]: DropdownSkeleton inconsistencies between design and code

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon#16430]
### Package

@carbon/react

### Package version

v11

### React version

1.57.0

### Description

DropdownSkeleton in React is missing an option for helper text, it also doesn't display the caret icon. (need to confirm with @carbon-design-system/design if this is a bug in code or in Figma)

**React**
<img width="351" alt="Screenshot 2024-05-10 at 7 54 26 AM" src="https://github.com/carbon-design-system/carbon/assets/2753488/50e63549-5d2e-4d89-aefa-4bdd0044562e">

**Figma**
![Screenshot 2024-05-10 at 7 53 25 AM](https://github.com/carbon-design-system/carbon/assets/2753488/b4f9b6b9-c893-4794-b177-635459669335)

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-dropdown--skeleton

### Steps to reproduce

Compare Figma to React

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by laurenmrice** [comment]
I think design needs to do an audit of our skeleton state specs for all components that have them to try to make them more consistent. I believe we may be moving away from including the icons in the skeleton state, so the code is probably more accurate for now.

**Comment by alisonjoseph** [comment]
@laurenmrice makes sense. Did we want to add the helper text option to this skeleton state?

**Comment by ra5enshuriken** [comment]
Hello Team,
I would like to contribute a fix towards this issue!

**Comment by Kritvi-bhatia17** [comment]
Hi @ra5enshuriken, thanks for showing interest! 
This task is part of a larger workstream where we’ll need to audit all skeleton states and establish a consistent pattern after team discussions. As a result, this issue isn’t currently open for contributions.

However there are plenty of other [issues that are open for contribution](https://github.com/carbon-design-system/carbon/issues?q=is:issue%20state:open%20label:%22needs:%20community%20contribution%22) and might align with your interests, feel free to explore them and pick what works best for you!
